core/storage/exploration/gae_models.py

```diff
@@ -865,6 +870,134 @@ def export_data(cls, user_id: str) -> Dict[str, List[str]]:
         }
 
 
+class TransientCheckpointUrlModel(base_models.BaseModel):
+    """Model for storing the progress of a logged-out user."""
+
+    # The exploration id.
+    exploration_id = (
+        datastore_services.StringProperty(required=True, indexed=True))
+    # The 6 digit unique id assigned to progress made by a logged-out user.
+    unique_progress_url_id = (
```

Comment:
We don't really store this id explicitly as it's the class id being stored as id, you should remove this explicit variable and use the class object id when required.

Reply:
Fixed. PTAL @paprajapati9